ui/src/admin/containers/AdminChronografPage.js

```diff
@@ -0,0 +1,383 @@
+    this.setState({showManageOverlay: true})
+  }
+  handleShowCreateUserOverlay = () => {
+    this.setState({showCreateUserOverlay: true})
+  }
+  handleHideOverlays = () => {
+    this.setState({showManageOverlay: false, showCreateUserOverlay: false})
+  }
+
+  handleFilterUsers = ({users, organization}) => {
+    const nextUsers = users || this.props.users
+    const nextOrganization = organization || this.props.currentOrganization
+
+    const filteredUsers =
+      nextOrganization.name === DEFAULT_ORG_NAME
+        ? nextUsers
+        : nextUsers.filter(
+            user =>
+              nextOrganization.name === NO_ORG
+                ? user.roles.length === 1 // Filter out if user is only part of Default Org
+                : user.roles.find(
+                    role => role.organization === nextOrganization.id
+                  )
+          )
+    this.setState({
+      filteredUsers,
+      organization: nextOrganization,
+      selectedUsers:
+        nextOrganization.name === DEFAULT_ORG_NAME
+          ? this.state.selectedUsers
+          : [],
+    })
+  }
+
+  handleToggleUserSelected = user => e => {
+    e.preventDefault()
+
+    const {selectedUsers} = this.state
+
+    const isUserSelected = selectedUsers.find(u => isSameUser(user, u))
+
+    const newSelectedUsers = isUserSelected
+      ? selectedUsers.filter(u => !isSameUser(user, u))
+      : [...selectedUsers, user]
+
+    this.setState({selectedUsers: newSelectedUsers})
+  }
+  handleToggleAllUsersSelected = areAllSelected => () => {
+    const {filteredUsers} = this.state
+
+    if (areAllSelected) {
+      this.setState({selectedUsers: []})
+    } else {
+      this.setState({selectedUsers: filteredUsers})
+    }
+  }
+
+  // SINGLE USER ACTIONS
+  handleCreateUser = user => {
+    const {links, actions: {createUserAsync}} = this.props
+    let newUser = user
+
+    if (
+      user.roles.length === 1 &&
+      user.roles[0].organization !== DEFAULT_ORG_ID
+    ) {
+      newUser = {
+        ...newUser,
+        roles: [
+          ...newUser.roles,
+          {organization: DEFAULT_ORG_ID, name: MEMBER_ROLE},
+        ],
+      }
+    }
+    createUserAsync(links.users, newUser)
+  }
+  // handleAddUserToOrg will add a user to an organization as a 'member'. if
+  // the user already has a role in that organization, it will do nothing.
+  handleAddUserToOrg = (user, organization) => {
+    const {actions: {updateUserAsync}} = this.props
+
+    updateUserAsync(user, {
+      ...user,
+      roles: [
+        ...user.roles,
+        {
+          name: MEMBER_ROLE, // TODO: remove this to let server decide when default org role is implemented
+          organization: organization.id,
+        },
+      ],
+    })
+  }
+  handleRemoveUserFromOrg = (user, organization) => {
+    const {actions: {updateUserAsync}} = this.props
+
+    let newRoles = user.roles.filter(r => r.organization !== organization.id)
+    if (newRoles.length === 0) {
```

Comment:
How would this happen? Can a user be removed from the Default Org?

Reply:
yes they can, but this code looks obsolete